language-plutus-core/test/Language/PlutusCore/PropTest.hs

```diff
@@ -83,15 +84,14 @@ toTyPropG typrop kG tyG = tyG_ok Cool.!=> typrop_ok
     tyG_ok    = checkClosedTypeG kG tyG
     typrop_ok = typrop kG tyG (toClosedType kG tyG)
 
-
 -- |Stream of type names t0, t1, t2, ..
 tynames :: [Text.Text]
 tynames = mkTextNameStream "t"
 
 
 -- |Convert type.
-toClosedType :: MonadQuote m
+toClosedType :: (MonadQuote m)
```

Comment:
Usually we either do all-mtl or all-explicit, so I'd probably put the `MonadError` constraint here instead of using `ExceptT` explicitly.